tamr_unify_client/models/attribute/resource.py

```diff
@@ -0,0 +1,48 @@
+from tamr_unify_client.models.attribute.type import AttributeType
+from tamr_unify_client.models.base_resource import BaseResource
+
+
+class Attribute(BaseResource):
+    """
+    A Unify Attribute.
+
+    See https://docs.tamr.com/reference#attribute-types
+    """
+
+    @classmethod
+    def from_json(cls, client, data, api_path):
```

Comment:
Note: unlike other resources, `api_path` is required, because there is not default collection of attributes.

Reply:
Should there be? That's a question for other people, so LGTM.

Reply:
Nope, attributes exist only in context. That context might be:
* A top-level dataset
* The type of another attribute
Attributes in different contexts can have the same local id (name) but different types etc.